Makefile

```diff
@@ -294,6 +301,15 @@ $(DIR)/.common: $(COMMON_DIR)
 	$(RM) -r $(DIR)/.common
 	cp -r $(COMMON_DIR) $(DIR)/.common
 
+
+# Helper function to re-format yamls using helper script
+define yamls_reformat
+	YQ=$(YQ) $(CURDIR)/hack/yq-fixup-yamls.sh $(1)
+endef
```

Comment:
I don't see this used anywhere. Please remove it or use it somehow.

Reply:
Wait a second.... now that I've read the whole PR, is there even a yq-fixup-yamls.sh script to call?
Let's keep this PR lean and drop everything related to YQ for now. Just do the yamllint bits and then in a later PR you can add YQ related stuff.

Reply:
right, in this version of the PE, I forgot to copy the script. ... I'll dropt the reformatting for now and we can discuss whether to add it later

Reply:
gone now